tests/roots/test-linkcheck-localserver-two-links/index.rst

```diff
@@ -1,6 +1,6 @@
-.. image:: http://localhost:7777/
+.. image:: http://localhost:7777/image.png
    :target: http://localhost:7777/
 
 `weblate.org`_
 
-.. _weblate.org: http://localhost:7777/
+.. _weblate.org: http://localhost:7777/weblate
```

Comment:
With all links being different, all items in the priority queue are now comparable (the uri is the second element of Hyperlink, right after the `next_check` and this change guarantees uris are unique in the queue). We can revert a1b8b1febb6ce0b7423bb78df30b0fc424c12325 :grin:

Reply:
I understand what this test does. Okay, let's remove (or deprecate) them all.